Formula/dotnet.rb

```diff
@@ -0,0 +1,33 @@
+class Dotnet < Formula
+  desc ".Net Core"
+  homepage "https://dotnet.microsoft.com/"
+  url "https://github.com/dotnet/source-build.git",
+      tag:      "v3.1.108-SDK",
+      revision: "05a0c8f6b13fcbec4b441c977b639b75e8a74f67"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "curl" => :build
```

Comment:
Can use use `curl` from macOS maybe?

Reply:
I tried for both `curl`, `icu4c` and `openssl`, but the build failed with `uses_from_macos` for all of them.